.github/workflows/build.yml

```diff
@@ -51,87 +53,123 @@ jobs:
       run: |
         pymol -ckqy testing/testing.py --run all
 
+
   build-Windows:
 
     runs-on: windows-latest
 
+    strategy:
+      matrix:
+        python-version: ["3.9", "3.10", "3.11", "3.12", "3.13"]
+
     env:
-      CONDA_ROOT: ${{github.workspace}}\..\tmp\miniforge
-      MINIFORGE_EXEC: ${{github.workspace}}\..\tmp\miniforge.exe
+      CONDA_ENV_NAME: "testing_env"
 
     steps:
-    - uses: actions/checkout@v4.0.0
-    - name: Download Miniforge
-      shell: cmd
-      run: |-
-        if not exist %CONDA_ROOT% mkdir %CONDA_ROOT%
-        curl -L -o %MINIFORGE_EXEC% https://github.com/conda-forge/miniforge/releases/download/24.11.0-0/Miniforge3-24.11.0-0-Windows-x86_64.exe
-        start /wait %MINIFORGE_EXEC% /S /D=%CONDA_ROOT%
+    - uses: actions/checkout@v4
 
-    - name: Set up Miniforge
-      shell: cmd
-      run: |-
-        CALL %CONDA_ROOT%\\Scripts\\activate.bat
-        conda install -y -c conda-forge -c schrodinger python cmake libpng freetype pyside6 glew libxml2 catch2=2.13.3 glm libnetcdf collada2gltf biopython msgpack-python pip python-build
+    - name: Install Miniforge
+      run: |
+        choco install miniforge3
 
-    - name: Conda info
-      shell: cmd
-      run: |-
-        CALL %CONDA_ROOT%\\Scripts\\activate.bat
-        conda info
+    - name: Add conda to PATH
+      run: |
+        echo "$env:CONDA" | Out-File -Append -FilePath $env:GITHUB_PATH
+        echo "$env:CONDA\Scripts" | Out-File -Append -FilePath $env:GITHUB_PATH
+
+    - name: Set up Miniforge
+      run: |
+        conda init powershell
+        conda create --name $env:CONDA_ENV_NAME -c conda-forge -c schrodinger `
+            python=${{ matrix.python-version }} `
+            pip `
+            catch2=2.13.3 `
+            cxx-compiler `
+            collada2gltf `
+            freetype `
+            glew `
+            glm `
+            libpng `
+            libxml2 `
+            libnetcdf `
+            libffi
```

Comment:
Although you linked the error and solution in the description. I'm still a bit unsure where this is coming from. I'm fairly confident this spawned from now not using conda-forge's Python

Reply:
You're right, I deleted this library in the last commit